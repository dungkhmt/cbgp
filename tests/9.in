50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
