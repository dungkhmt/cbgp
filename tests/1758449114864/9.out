(31, 48), (42, 38), (45, 23), (23, 47), (42, 33), (44, 7), (33, 3), (32, 43), (5, -1), (27, 3), (35, 6), (35, 12), (16, 4), (45, 10), (35, 8), (41, 5), (14, 7), (19, 0), (3, 4), (3, 5), (7, 4), (17, 16), (34, 10), (11, 16), (46, 10), (37, 19), (44, 13), (48, 3), (40, 45), (29, 20), (5, 1), (30, 0), (21, 15), (19, 26), (7, 8), (22, 16), (30, 11), (28, 23), (12, 25), (47, 38), (28, 2), (24, 18), (21, 1), (29, 18), (11, 43), (44, 9), (39, 15), (30, 20), (36, 11), (0, 1), (44, 2), (13, 2), (36, -1), (48, 1), (41, 21), (34, 13), (14, 8), (32, 16), (4, 33), (41, 15), (12, 0), (7, 0), (13, 10), (41, 26), (18, -1), (44, 6), (5, 0), (28, 10), (29, 24), (22, 36), (38, 31), (31, 3), (20, 18), (17, 22), (16, -1), (30, 18), (30, -1), (45, 28), (1, 2), (22, 11), (25, 6), (42, 31), (47, 10), (12, 8), (41, 19), (25, 2), (34, 46), (8, 6), (40, 47), (44, 8), (32, 30), (38, 1), (32, 7), (32, 11), (12, 6), (48, 38), (37, 41), (22, -1), (45, 47), (31, 33), 
numberIntersection=65 
minAngle=0.02912203765677024 
sumAngle=48.353002842590385 
minEdgeLength=2.23606797749979 
sumEdgeLength=4584.1696950179985 
minDistance=0.018242223059788013 
sumDistance=305312.8154192394 

-65.0000000000 0.0291220377 48.3530028426 2.2360679775 4584.1696950180 0.0182422231 305312.8154192394
0: (89, 30),
1: (80, 50),
2: (44, 65),
3: (39, 13),
4: (23, 82),
5: (85, 47),
6: (36, 0),
7: (58, 196),
8: (0, 25),
9: (46, 45),
10: (81, 200),
11: (95, 62),
12: (66, 4),
13: (44, 74),
14: (10, 67),
15: (74, 124),
16: (69, 66),
17: (59, 71),
18: (76, 2),
19: (83, 69),
20: (97, 14),
21: (72, 123),
22: (56, 74),
23: (82, 137),
24: (91, 19),
25: (58, 12),
26: (83, 74),
27: (38, 31),
28: (108, 125),
29: (97, 42),
30: (104, 10),
31: (42, 31),
32: (173, 34),
33: (6, 37),
34: (184, 29),
35: (37, 4),
36: (60, 74),
37: (85, 72),
38: (49, 46),
39: (77, 167),
40: (78, 152),
41: (94, 111),
42: (39, 41),
43: (135, 49),
44: (25, 76),
45: (83, 121),
46: (187, 72),
47: (77, 168),
48: (53, 28),
49: (200, 200),
89 30 80 50 44 65 39 13 23 82 85 47 36 0 58 196 0 25 46 45 81 200 95 62 66 4 44 74 10 67 74 124 69 66 59 71 76 2 83 69 97 14 72 123 56 74 82 137 91 19 58 12 83 74 38 31 108 125 97 42 104 10 42 31 173 34 6 37 184 29 37 4 60 74 85 72 49 46 77 167 78 152 94 111 39 41 135 49 25 76 83 121 187 72 77 168 53 28 200 200 
0: (89, 30), 1: (80, 50), 2: (44, 65), 3: (39, 13), 4: (23, 82), 5: (85, 47), 6: (36, 0), 7: (58, 196), 8: (0, 25), 9: (46, 45), 10: (81, 200), 11: (95, 62), 12: (66, 4), 13: (44, 74), 14: (10, 67), 15: (74, 124), 16: (69, 66), 17: (59, 71), 18: (76, 2), 19: (83, 69), 20: (97, 14), 21: (72, 123), 22: (56, 74), 23: (82, 137), 24: (91, 19), 25: (58, 12), 26: (83, 74), 27: (38, 31), 28: (108, 125), 29: (97, 42), 30: (104, 10), 31: (42, 31), 32: (173, 34), 33: (6, 37), 34: (184, 29), 35: (37, 4), 36: (60, 74), 37: (85, 72), 38: (49, 46), 39: (77, 167), 40: (78, 152), 41: (94, 111), 42: (39, 41), 43: (135, 49), 44: (25, 76), 45: (83, 121), 46: (187, 72), 47: (77, 168), 48: (53, 28), 49: (200, 200), 
numberIntersection=65 minAngle=0.02912203765677024 sumAngle=48.353002842562596 minEdgeLength=2.23606797749979 sumEdgeLength=4584.169695017995 minDistance=0.018242223059788013 sumDistance=305312.81541923765 
Time taken: 37.302 s
